### PR TITLE
Add flask migrate

### PR DESCRIPTION
**Pull request by jonascampos1**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -12,11 +12,22 @@ services:
       - POSTGRES_USER=jonas
       - POSTGRES_HOST=db
       - POSTGRES_PORT=5432
+      - POSTGRES_DB=jonas
   point_of_sale:
-    restart: always
```
Comment (rrgarciach) [inline docker-compose.yml:16]: put this back and set it as `on-failure`

File: models/user.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                  Column("id", INTEGER, primary_key=True),
+                  Column("username", VARCHAR(150)),
+                  Column("password", SMALLINT),
+                  Column("salt", VARCHAR(45)),
+                  Column("created_at", DateTime),
+                  Column("updated_at", DateTime),
+                  Column("deleted_at", DateTime)
+                )
+
+#meta.create_all(engine)
```
Comment (rrgarciach) [inline models/user.py]: is it still useful? if not, let's remove any unused commented line

File: models/user.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+class UserM(BaseModel):
```
Comment (rrgarciach) [inline models/user.py]: Let's better simply call it `User` as it's already implied that it is a model

File: setup.py [hunk 4]
```diff
@@ -1,20 +1,30 @@
...
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://jonas:password@localhost:5432/jonas'
+CORS(app)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://'+os.getenv('POSTGRES_USER')+':'+os.getenv('POSTGRES_PASSWORD')+'@db:5432/jonas'
 db = SQLAlchemy(app)
 migrate = Migrate(app, db, directory='config/db/postgres/migrations')
 
-class User(db.Model):
+class Users(db.Model):
```
Comment (rrgarciach) [inline setup.py:21]: Why did this change from singular to plural?

File: setup.py [hunk 5]
```diff
@@ -1,20 +1,30 @@
...
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(128))
+    username = db.Column(VARCHAR(16), nullable=False)
+    password = db.Column(VARCHAR(64), nullable=False)
+    salt = db.Column(VARCHAR(64), nullable=False)
+    created_at = db.Column(DateTime, nullable=False, server_default=text('NOW()'))
+    updated_at = db.Column(DateTime, nullable=False, onupdate=text('NOW()'))
+    deleted_at = db.Column(DateTime, nullable=True)
+    test = db.Column(VARCHAR(10), nullable=True)
```
Comment (rrgarciach) [inline setup.py:29]: what's this column meant for?

File: controllers/auth.py [hunk 6]
```diff
@@ -1,5 +1,83 @@
...
+
+from services.auth import auth_user
+import hashlib
+from datetime import datetime, timezone
+import secrets
+
+
+def controller(app):
+    @app.route('/', methods=["GET"])
+    def start():
```
Comment (rrgarciach) [inline controllers/auth.py]: what's te purpose of this `start` function?

File: controllers/auth.py [hunk 7]
```diff
@@ -1,5 +1,83 @@
...
+        return "Online"
+
+    route_api='/api/v1/'
+
+"""    @app.route(route_api+'users', methods=["GET"])
```
Comment (rrgarciach) [inline controllers/auth.py]: remove any unused commented line

**Review by rrgarciach — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jonascampos1) [reply docker-compose.yml:16]: Ok

File: setup.py (hunk 4 above)
Comment (jonascampos1) [reply setup.py:21]: In PostgreSQL by default exists a  reserverd word called user, this is the reason why i changed to users

File: setup.py (hunk 5 above)
Comment (jonascampos1) [reply setup.py:29]: Only test purpose, but i need to delete!

File: controllers/auth.py (hunk 6 above)
Comment (jonascampos1) [reply controllers/auth.py]: Only test if the service is online but, i need to delete! Is irrelevant for the api functionallity

File: models/user.py (hunk 2 above)
Comment (jonascampos1) [reply models/user.py]: The command was used when the table does not exists in DB, but is unuseful when the table is created trought the migration. I will delete

**Pull request closed without merging**